client/packages/system/src/Patient/ProgramEnrolment/Components/DetailModal.tsx

```diff
@@ -48,11 +48,17 @@ const useUpsertProgramEnrolment = (
         });
 };
 
-export const ProgramDetailModal: FC = () => {
+export const ProgramDetailModal = ({
+  patientId: inputPatientId,
+}: {
+  patientId?: string;
+}) => {
   const t = useTranslation();
-  const patientId = usePatient.utils.id();
```

Comment:
Had to pass patientId into component when used here, as it can no longer fetch patientID from URL.

Still uses `usePatient.utils.id()` if it can (below), so I don't have to update the other usage(s) of the Form.

Personally, I think it's a bit dodgy to rely on the presence of a URL parameter in a Modal, as they can easily be used in places other than their original implementation (as is the case here!)